scripts/vagrantCloudBasedContainers.sh

```diff
@@ -56,4 +56,4 @@ curl --retry $COUCHDB_HEALTHCHECK_TIMEOUT --retry-delay 1 --retry-connrefused ht
 
```

Comment:
curl seems to fail at the first attempt if a ["connection reset by peer"](https://ci.gpii.net/job/universal-node-production-tests/700/console) is raised, wget does it better:

`wget --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 -t 30 http://localhost:$COUCHDB_PORT`

Reply:
Thanks for reviewing this PR. I couldn't reproduce the issue you mentioned here. It seems the PR you linked to might not be using F27 and/or contain the new healtcheck delay that was increased here. 

My test results:

```
Digest: sha256:b95dce63ab64991640e5c9d4cc1597055690b1c1bb79ab30829d498f5f2301fc
Status: Downloaded newer image for docker.io/couchdb:latest
3a796c6a2f3aa371bd3c439b96a1fb7d5ee3e305400952c87c27ce6d88d57895
+ sleep 10



+ curl --retry 30 --retry-delay 1 --retry-connrefused http://localhost:5984
{"couchdb":"Welcome","version":"2.1.1","features":["scheduler"],"vendor":{"name":"The Apache Software Foundation"}}
+ docker run -d -p 9081:9081 --name preferences --link couchdb -e NODE_ENV=gpii.config.preferencesServer.standalone.production -e GPII_PREFERENCES_DATASOURCE_URL=http://couchdb:5984/preferences/%userToken -e GPII_PREFERENCES_LISTEN_PORT=9081 vagrant-universal
```

I agree wget handles this better. It seems curl is making assumptions about what a "transient" error mean that are not valid for our environment. I opened an issue in curl/curl repo to discuss this (2350).

If this issue is not introduced by this PR specifically, would it be better to track this in a separate JIRA?

Reply:
Yes, this change doesn't cover the same issue that this PR covers. I guess that we can follow up with this conversation here [GPII-2885](https://issues.gpii.net/browse/GPII-2885)